src/AutoRest.CSharp/Output/Models/Types/OutputLibrary.cs

```diff
@@ -331,7 +344,17 @@ private void DecorateOperationGroup()
                 {
                     operationsGroup.Resource = nameOverride;
                 }
+
             }
+            //now that we have resolved all operations groups to resource types above, can try to solve for the parent
+            foreach (var operationsGroup in _codeModel.OperationGroups)
+            {
+                if (operationsGroup.Parent != null && !ResourceTypes.Contains(operationsGroup.Parent))
```

Comment:
not sure this is ever possible since line 338 always adds if it wasn't null

Reply:
it is check "parent" null not "Parent"

So if it was overridden, add it. If it was not, verify it was actually in the swagger